ui/raidboss/data/06-ew/raid/p6s.ts

```diff
@@ -48,6 +80,51 @@ const triggerSet: TriggerSet<Data> = {
       netRegex: NetRegexes.startsUsing({ id: '7881', source: 'Hegemone', capture: false }),
       response: Responses.goSides(),
     },
+    {
+      id: 'P6S Exchange of Agonies Markers',
+      type: 'HeadMarker',
+      netRegex: NetRegexes.headMarker({}),
+      condition: (data, matches) => {
+        return data.me === matches.target && !(/(00(?:4F|5[0-6]))|013E|0148|0065/).test(getHeadmarkerId(data, matches));
```

Comment:
I'm not sure why you need a test here too.

What about: (1) condition is just data.me===matches.target and (2) drop the unknown from the switch statement and just return silently if it's not one of the correct markers?

Reply:
It looked like I was missing some headmarkers so I put an unknown catch, but your previous comment should mean we have them all now.